<launch>
    <!-- parameters -->
    <arg name="vertex_size" default = "8" /> 
    
    <!-- ra_node -->
    <node name="ra_node" pkg="ra" type="ra_node" output="screen">
        <param name="vertex_size" type="double" value="$(arg vertex_size)" />
    </node>

    <!-- rviz -->
    <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find ra)/rviz/ra.rviz" />
</launch>

